noxfile.py

```diff
@@ -644,14 +667,16 @@ def _cmake(session, build_type):
     ]
     if IS_WINDOWS:
         build_args.extend(["-G", "MinGW Makefiles"])
+    if IS_MACOS:
+        build_args.append("-DCMAKE_OSX_DEPLOYMENT_TARGET:STRING=13.0")
```

Comment:
This is temporary! The default value used by CMake is whatever the current version is on the machine. Here that's 13.6 as of https://github.com/actions/runner-images/blob/3793317199f7f1eebff21ae96c77878546f06389/images/macos/macos-13-Readme.md.

However, only 13.0 is allowed, e.g.:

```
$ python3.11 -m pip debug --verbose | grep cp311-cp311-macosx_13
  cp311-cp311-macosx_13_0_arm64
  cp311-cp311-macosx_13_0_universal2
```